### PR TITLE
feat: reject streetname underlying addresses

### PR DESCRIPTION
**Pull request by emalfroy**

### CODE REVIEW

File: src/AddressRegistry.Projections.Extract/AddressExtract/AddressExtractProjectionsV2.cs [hunk 1]
```diff
@@ -174,6 +174,27 @@ public AddressExtractProjectionsV2(IOptions<ExtractConfig> extractConfig, Encodi
                 UpdateVersie(item, message.Message.Provenance.Timestamp);
             });
 
+            When<Envelope<AddressWasRejectedBecauseHouseNumberWasRejected>>(async (context, message, ct) =>
```
Comment (jvandaal) [inline src/AddressRegistry.Projections.Extract/AddressExtract/AddressExtractProjectionsV2.cs:177]: You are now handling this event twice. 
image

File: test/AddressRegistry.Tests/ProjectionTests/Legacy/AddressDetailItemV2Tests.cs [hunk 2]
```diff
@@ -280,6 +280,71 @@ await Sut
...
+                { AddEventHashPipe.HashMetadataKey, addressWasProposedV2.GetHash() }
+            };
+
+            var addressWasApproved = _fixture.Create<AddressWasApproved>();
+            var approveMetadata = new Dictionary<string, object>
+            {
+                { AddEventHashPipe.HashMetadataKey, addressWasApproved.GetHash() }
+            };
+
+            var addressWasRejected = _fixture.Create<AddressWasRetiredBecauseStreetNameWasRejected>();
```
Comment (jvandaal) [inline test/AddressRegistry.Tests/ProjectionTests/Legacy/AddressDetailItemV2Tests.cs:327]: rename to addressWasRetired

File: test/AddressRegistry.Tests/ProjectionTests/Legacy/AddressListProjectionsV2Tests.cs [hunk 3]
```diff
@@ -326,6 +326,71 @@ await Sut
...
+                { AddEventHashPipe.HashMetadataKey, addressWasProposedV2.GetHash() }
+            };
+
+            var addressWasApproved = _fixture.Create<AddressWasApproved>();
+            var approveMetadata = new Dictionary<string, object>
+            {
+                { AddEventHashPipe.HashMetadataKey, addressWasApproved.GetHash() }
+            };
+
+            var addressWasRejected = _fixture.Create<AddressWasRetiredBecauseStreetNameWasRejected>();
```
Comment (jvandaal) [inline test/AddressRegistry.Tests/ProjectionTests/Legacy/AddressListProjectionsV2Tests.cs:373]: Rename to addressWasRetired

File: test/AddressRegistry.Tests/ProjectionTests/Legacy/AddressWfsProjectionsTests.cs [hunk 4]
```diff
@@ -271,6 +271,69 @@ await Sut
...
+                { AddEventHashPipe.HashMetadataKey, addressWasProposedV2.GetHash() }
+            };
+
+            var addressWasApproved = _fixture.Create<AddressWasApproved>();
+            var approveMetadata = new Dictionary<string, object>
+            {
+                { AddEventHashPipe.HashMetadataKey, addressWasApproved.GetHash() }
+            };
+
+            var addressWasRejected = _fixture.Create<AddressWasRetiredBecauseStreetNameWasRejected>();
```
Comment (jvandaal) [inline test/AddressRegistry.Tests/ProjectionTests/Legacy/AddressWfsProjectionsTests.cs:317]: Rename to addressWasRetired.

**Review by ArneD — APPROVED** [review]

**Pull request merged**